torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -3588,6 +3588,33 @@ def _call_function(
             tx, user_func, user_args, kwargs, self.value._name, subgraph_name="subgraph"
         )
 
+        # Validate
+        expected_num_inputs = len(in_placements.value)
+        actual_num_inputs = len(body_gmod.graph.find_nodes(op="placeholder"))
+        expected_num_outputs = len(out_placements.value)
+        assert len(body_gmod.graph.find_nodes(op="output")) == 1
```

Comment:
just curious, was there a bug at some point that would have failed this assert? (I feel like this invariant has been around for a long time)